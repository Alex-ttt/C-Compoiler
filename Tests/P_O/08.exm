Wrong expression	 line 1, col 8
