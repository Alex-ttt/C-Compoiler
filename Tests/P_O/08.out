Wrong expression	 line 2, col 20

